spec/truffle.mspec

```diff
@@ -152,6 +152,10 @@ class MSpecScript
   MSpec.enable_feature :readline
 
   set :files, get(:command_line) + get(:language) + get(:core) + get(:library) + get(:truffle) + get(:security)
+
+  # All specs, including specs needing C-extensions support.
+  # 2.4/2.5 specs are not included as they need to run in a separate process.
```

Comment:
So, not "all" then? :-)

Reply:
"All we can run in a single process". Any idea of a better name?

Reply:
`:all_fast`, `:all_single_process`, `:most`? I'm not really sure. But I think at some point someone is going to be surprised to learn that `all` doesn't include everything.

Reply:
"all specs of the Ruby version we officially support" seems fair enough.
I think the comment is clear enough in what `:all` means.